app/views/assignments/peer_review.html.erb

```diff
@@ -0,0 +1,324 @@
+<%= render partial: 'peer_submissions_table', formats: [:'js.jsx'], handlers: [:erb] %>
+<script>
```

Comment:
@david-yz-liu I did not know students were not going to be allowed to create groups. I will make the necessary changes including those suggested below. Thank you.